xwiki-commons-core/xwiki-commons-xml/src/main/java/org/xwiki/xml/html/HTMLUtils.java

```diff
@@ -134,7 +134,7 @@ public String escapeElementEntities(String text)
                 }
             } else {
                 result = escapeAmpersand(text);
-                StringUtils.replaceEach(text, REPLACE_ELEMENTS_SEARCH, REPLACE_ELEMENTS_RESULT);
+                result = StringUtils.replaceEach(result, REPLACE_ELEMENTS_SEARCH, REPLACE_ELEMENTS_RESULT);
```

Comment:
This looks like a bug fix unrelated to the HTMLCleaner upgrade.

Reply:
It's not directly related to the upgrade, but it's related to the fix of escaped values. I put everything on the same basket here.